package.json

```diff
@@ -27,7 +27,7 @@
     "debug": "^4.3.1",
     "ec-pem": "^0.18.0",
     "jsonwebtoken": "^8.5.1",
-    "jsp-raknet": "github:extremeheat/raknet#client",
+    "jsp-raknet": "latest",
```

Comment:
wouldnt it be better to stick to a major ver?

Reply:
Fixed, I don’t think the js raknet implementation works properly at the moment (also we don’t test against it), so it could be removed also but I guess it can be kept for now.